aggregators/will-get-add-to-homescreen-prompt/index.js

```diff
@@ -25,20 +25,39 @@ class AddToHomescreen extends Aggregate {
     return 'Will Get Add to Homescreen Prompt';
   }
 
+  /**
+   * For the install-to-homescreen / install-app-banner prompt to show,
+   * Chrome needs the following:
+   *   - has a registered SW
```

Comment:
yup. https://developers.google.com/web/updates/2015/03/increasing-engagement-with-app-install-banners-in-chrome-for-android?hl=en